lib/cyclonedx/report_rust_crates_cyclonedx.rb

```diff
@@ -3,5 +3,14 @@ class ReportRustCrates < Base
     def initialize(scan_report)
       super(scan_report)
     end
+
+    def package_url(dependency)
+      "pkg:#{dependency[:type]}/#{dependency[:name]}@#{version_string(dependency)}"
+    end
+
+    # Return version string to be used in purl or component
+    def version_string(dependency)
+      (dependency[:version_tag]).to_s
```

Comment:
Will the version tag always be `@x.x.x` ?

Reply:
minor nit you don't need the () here

Reply:
I think so it seems like rust follows semantic versioning 

 http://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/cargo/reference/manifest.html#:~:text=Use%20version%20numbers%20with%20three%20numeric%20parts%20such%20as%201.0.0%20rather%20than%201.0.